content/en/docs/components/pipelines/sdk/connect-api.md

```diff
@@ -1,255 +1,359 @@
 +++
-title = "Connecting to Kubeflow Pipelines using the SDK client"
-description = "How to connect to Kubeflow Pipelines using the SDK client and configure the SDK client using environment variables"
+title = "Connect the Pipelines SDK to Kubeflow Pipelines"
+description = "How to connect the Pipelines SDK to Kubeflow Pipelines in various ways"
 weight = 25
 +++
 
-This guide demonstrates how to connect to Kubeflow Pipelines using [the Kubeflow Pipelines SDK client](/docs/components/pipelines/sdk/sdk-overview/), and how to [configure the SDK client using environment variables](#configure-sdk-client-by-environment-variables).
+How you connect the Pipelines SDK to Kubeflow Pipelines will depend on what type of Kubeflow deployment you have, and where you are running your code:
 
+* [Standalone Kubeflow (from inside cluster)](#standalone-kubeflow-from-inside-cluster)
+* [Standalone Kubeflow (from outside cluster)](#standalone-kubeflow-from-outside-cluster)
+* [Multi-user Kubeflow (from inside cluster)](#multi-user-kubeflow-from-inside-cluster)
+* [Multi-user Kubeflow (from outside cluster)](#multi-user-kubeflow-from-outside-cluster)
 
-The Kubeflow Pipelines REST API is available at the same endpoint as the Kubeflow Pipelines user interface (UI).
-The SDK client can send requests to this endpoint to upload pipelines, create pipeline runs, schedule recurring runs, and more.
+## Standalone Kubeflow (from inside cluster)
 
+{{% alert title="Warning" color="warning" %}}
+This will only work for standalone Kubeflow Pipelines, for multi-user Kubeflow see [Multi-user Kubeflow (from inside cluster)](#multi-user-kubeflow-from-inside-cluster).
+{{% /alert %}}
```

Comment:
It's for visual clarity, and to prevent people blindly copying the code in the "standalone" sections and it not working in a "full" kubeflow deployment.